.github/workflows/test-publish.yaml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   publish:
-    if: github.event.pull_request.head.repo.full_name == github.repository
+    if: ${{ github.repository == 'getsentry/sentry-android-gradle-plugin'}}
```

Comment:
right now it causes:

> caused by: sentry reported an error: Invalid token header. No credentials provided. (http status: 401)

we want to skip it in such cases, right? and not fail.

Reply:
Yes and in such cases `github.repository` should be `'cortinico/sentry-android-gradle-plugin'`.
Seems like it's not the case actually